frontend/public/components/network-policy.jsx

```diff
@@ -178,26 +184,28 @@ const IngressRow = ({ ingress, namespace, podSelector }) => {
 };
 
 const Details_ = ({ obj: np, flags }) => {
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text="Namespace Details" />
+        <SectionHeading text={t('network-policy~Namespace details')} />
         <ResourceSummary resource={np} podSelector={'spec.podSelector'} showPodSelector />
       </div>
       <div className="co-m-pane__body">
-        <SectionHeading text="Ingress Rules" />
+        <SectionHeading text={t('network-policy~Ingress rules')} />
         <p className="co-m-pane__explanation">
-          Pods accept all traffic by default. They can be isolated via Network Policies which
-          specify a whitelist of ingress rules. When a Pod is selected by a Network Policy, it will
-          reject all traffic not explicitly allowed via a Network Policy. See more details in{' '}
+          {t(
+            'network-policy~ingress_rules_description',
+            'Pods accept all traffic by default. They can be isolated via Network Policies which specify a whitelist of ingress rules. When a Pod is selected by a Network Policy, it will reject all traffic not explicitly allowed via a Network Policy. See more details in:',
+          )}
```

Comment:
I'm concerned about defined keys here as well.

Reply:
@spadgett & @rebeccaalpert, I think we need a ruling here :-)  I'm concerned other devs might be using keys.  We had a thread about their usage, and I thought it was best practice to use the key for long descriptions.

Reply:
I see some keys being used in @rhamilto 's PR [here](https://github.com/openshift/console/pull/6883/files#diff-938bd1f94fe7531ac8316dbfa825e7947b180102cdbd171e53c8758af149030fR20).

Reply:
Yeah, I haven't reviewed Robb's yet. It's up to Sam ultimately; I'm just concerned about the impact on translation.

Reply:
Changed to use full message as the key.